pkg/segment/tracing/handler/tracehandler.go

```diff
@@ -150,6 +151,66 @@ func ProcessSearchTracesRequest(ctx *fasthttp.RequestCtx, myid uint64) {
 	ctx.SetStatusCode(fasthttp.StatusOK)
 }
 
+func ProcessTotalTracesRequest(ctx *fasthttp.RequestCtx, myid uint64) {
```

Comment:
It looks like this function repeats a lot of the work done by the `ProcessTotalTracesRequest` function. Would it be better to calculate this result in the `ProcessTotalTracesRequest` function and send the total count as another json parameter, so that then we don't have this duplicated code and it would probably save the backend some duplicate work? Or would that not work for some reason?